main/res/values/strings.xml

```diff
@@ -1025,6 +1029,8 @@
     <string name="waypoint_duplicate">Duplicate waypoint</string>
     <string name="waypoint_duplicated">Waypoint duplicated</string>
     <string name="waypoint_copy_of">Copy of</string>
+    <string name="waypointcalc_notes">Notes:</string>
```

Comment:
Probably don't even need the labels at all.  The 'Result' field is fairly self explanatory and the hint text in the 'Notes' field should be enough to indicate what it is:
![screenshot_1500720039](https://user-images.githubusercontent.com/6573522/28490481-2ce57448-6f1a-11e7-978c-4161a96dc5e1.png)